### PR TITLE
fix(planner/python):  Python DetermineFramework

### PR DESCRIPTION
**Pull request by ScmTble**

### CODE REVIEW

File: internal/python/plan_test.go [hunk 1]
```diff
@@ -355,22 +355,8 @@ func TestHasDependency_Pipenv_WithObsoleteRequirements(t *testing.T) {
...
-	_ = afero.WriteFile(fs, "Pipfile.lock", []byte("foo"), 0o644)
-	_ = afero.WriteFile(fs, "requirements.txt", []byte("bar"), 0o644)
-
-	ctx := &pythonPlanContext{
-		Src:            fs,
-		PackageManager: optional.Some(types.PythonPackageManagerPipenv),
-	}
-
-	assert.True(t, HasDependency(ctx, "foo"))
-	assert.False(t, HasDependency(ctx, "bar"))
```
Comment (pan93412) [inline internal/python/plan_test.go:373]: Why removes this test?

File: internal/python/plan_test.go [hunk 2]
```diff
@@ -276,8 +276,8 @@ func TestHasDependency_Unknown(t *testing.T) {
 	}
 
 	// should always False
-	assert.False(t, HasDependency(ctx, "foo"))
-	assert.False(t, HasDependency(ctx, "bar"))
```
Comment (pan93412) [inline internal/python/plan_test.go:280]: Don't modify the current test. You can copy a new function based on your new `HasDependencyWithFile` with `TestHashDependencyWithFile_` prefixed.

**Review by pan93412 — CHANGES_REQUESTED** [review]

File: internal/python/plan_test.go (hunk 1 above)
Comment (ScmTble) [reply internal/python/plan_test.go:373]: check Dependency in Pipfile not in Pipfile.lock TestHasDependency_PipenvDep has check

Comment (pan93412) [reply internal/python/plan_test.go:373]: 我觉得留着也没关系，或许有人依赖这个行为 🤔  如果要删的话，我印象中我还有 `poetry.lock` 的对应测试，这部份也可能需要你检查一下～

**Review by pan93412 — APPROVED** [review]

**Pull request merged**